package-parser/package_parser/processing/annotations/_generate_annotations.py

```diff
@@ -224,8 +231,16 @@ def __get_parameter_info(qname: str, usages: UsageCountStore) -> ParameterInfo:
     :param usages: UsageStore
     :return ParameterInfo
     """
-
-    values = [(it[0], it[1]) for it in usages.value_usages[qname].items() if it[1] > 0]
+    # Creates a list of tuples with values value_name and value_total_usages
+    values = []
+    for it in usages.value_usages[qname].items():
```

Comment:
Destructure the tuple:

```py
for stringifiedValue, count in usages.value_usages[qname].items():
```

Afterwards use `stringifiedValue` and `count` in place of `it[0]` and `it[1]`. This makes the code much easier to read.